jonas/desktop/custom.nix

```diff
@@ -0,0 +1,14 @@
+{ config, pkgs, ... }:
+
+{
+  home.packages = with pkgs; [
+    (writeShellScriptBin "blender" ''
+      ${nixgl.nixGLIntel}/bin/nixGLIntel ${blender}/bin/blender
+    '')
+
+
+    (writeShellScriptBin "globalprotect" ''
+      ${nixgl.nixGLIntel}/bin/nixGLIntel ${globalprotect-openconnect}/bin/gpclient
```

Comment:
add $@ to the end, that passes all CLI arguments to gpclient (resign from Viaplay already)

Reply:
lol